### PR TITLE
Dev tl yz

### PR DESCRIPTION
**Pull request by yz540**
Please review the ReadMe and the code.

### CODE REVIEW

File: ros/src/tl_detector/light_classification/tl_classifier.py [hunk 1]
```diff
@@ -1,19 +1,116 @@
...
+            boxes = np.squeeze(boxes)
+            scores = np.squeeze(scores)
+            classes = np.squeeze(classes)
+        
+            confidence_cutoff = 0.7
+            # Filter boxes with a confidence score less than `confidence_cutoff`
+            boxes, scores, classes = self.filter_boxes(confidence_cutoff, boxes, scores, classes)
+        
+        
+        if len(classes)>0:
```
Comment (bitsurgeon) [inline ros/src/tl_detector/light_classification/tl_classifier.py]: please considering align the class id and the light state

- current trained model will have label - state as [1-green, 2-red, 3-yellow]
- when the list is empty, it is unknown

File: ros/src/tl_detector/light_classification/tl_classifier.py [hunk 2]
```diff
@@ -1,19 +1,116 @@
...
+        
+            self.image_tensor = self.graph.get_tensor_by_name('image_tensor:0')
+            self.detection_boxes = self.graph.get_tensor_by_name('detection_boxes:0')
+            self.detection_scores = self.graph.get_tensor_by_name('detection_scores:0')
+            self.detection_classes = self.graph.get_tensor_by_name('detection_classes:0')
+        rospy.loginfo("loaded graph............")
+        self.sess = tf.Session(graph=self.graph)
+
+
+    def to_image_coords(self, boxes, height, width):
```
Comment (bitsurgeon) [inline ros/src/tl_detector/light_classification/tl_classifier.py]: we may don't need this function as only the class is and/or scores would be the outputs

File: ros/src/tl_detector/light_classification/tl_classifier.py [hunk 3]
```diff
@@ -1,19 +1,116 @@
...
+        box_coords = np.zeros_like(boxes)
+        box_coords[:, 0] = boxes[:, 0] * height
+        box_coords[:, 1] = boxes[:, 1] * width
+        box_coords[:, 2] = boxes[:, 2] * height
+        box_coords[:, 3] = boxes[:, 3] * width
+        
+        return box_coords
+
+
+    def draw_boxes(self, image, boxes, classes, scores):
```
Comment (bitsurgeon) [inline ros/src/tl_detector/light_classification/tl_classifier.py]: same as above, we may no need to draw the images/boxes

File: ros/src/tl_detector/light_classification/tl_classifier.py (hunk 2 above)
Comment (yz540) [reply ros/src/tl_detector/light_classification/tl_classifier.py]: removed in the next push

File: ros/src/tl_detector/light_classification/tl_classifier.py (hunk 3 above)
Comment (yz540) [reply ros/src/tl_detector/light_classification/tl_classifier.py]: removed in the next push

File: ros/src/tl_detector/light_classification/tl_classifier.py (hunk 1 above)
Comment (yz540) [reply ros/src/tl_detector/light_classification/tl_classifier.py]: corrected in the next push

**Pull request merged**